packages/forma-36-react-components/src/components/Asset/README.mdx

```diff
@@ -0,0 +1,144 @@
+import { Asset } from './Asset';
+
+# Asset
+
+The Asset component is a visual representation of an asset file such as a video or a PDF document.
+
+<Asset title="Archive of files" type="archive" />
+
+## Table of contents
+
+- [Variations](#variations)
+  - [Archive](#archive)
+  - [Audio](#audio)
+  - [Code](#code)
+  - [Image](#image)
+  - [Markup](#markup)
+  - [PDF](#pdf)
+  - [Plaintext](#plaintext)
+  - [Presentation](#presentation)
+  - [Rich text](#rich-text)
+  - [Spreadsheet](#spreadsheet)
+  - [Video](#video)
+- [Usage](#usage)
+  - [An image asset and an archived image asset](#an-image-asset-and-an-archived-image-asset)
+- [An archive asset](#an-archive-asset)
+
+## Variations
+
+### Archive
+
+<Asset
+  src="https://placekitten.com/200/300"
+  title="An archive asset"
+  type="archive"
+/>
+
+### Audio
+
+<Asset
+  src="https://placekitten.com/200/300"
+  title="An audio asset"
+  type="audio"
+/>
+
+### Code
+
+<Asset src="https://placekitten.com/200/300" title="A code asset" type="code" />
+
+### Image
+
+<Asset
+  src="https://placekitten.com/200/300"
+  title="An image asset"
+  type="image"
+/>
+
+### Markup
+
+<Asset
+  src="https://placekitten.com/200/300"
+  title="A markup asset"
+  type="markup"
+/>
+
+### PDF
+
+<Asset src="https://placekitten.com/200/300" title="A PDF asset" type="pdf" />;
+
+### Plaintext
+
+<Asset
+  src="https://placekitten.com/200/300"
+  title="A plaintext asset"
+  type="plaintext"
+/>
+
+### Presentation
+
+<Asset
+  src="https://placekitten.com/200/300"
+  title="A presentation asset"
+  type="presentation"
+/>
+
+### Rich Text
+
+<Asset
+  src="https://placekitten.com/200/300"
+  title="A rich text asset"
+  type="richtext"
+/>
+
+### Spreadsheet
+
+<Asset
+  src="https://placekitten.com/200/300"
+  title="A spreadsheet asset"
+  type="spreadsheet"
+/>
+
+### Video
+
+<Asset
+  src="https://placekitten.com/200/300"
+  title="A video asset"
+  type="video"
+/>
+
+## Usage
+
+```jsx
+import { Asset } from '@contentful/forma-36-react-components';
+
+<Asset
+  src="https://placekitten.com/200/300"
+  title="An archive asset"
+  type="archive"
+/>;
+```
+
+### An image asset and an archived image asset
+
+Image assets are rendered directly unless they have the prop `status="archived"`.
```

Comment:
@denkristoffer I find it a bit unclear. It would find it a bit more useful if it would more descriptive. Something around the fact that if the status is archived instead of the image the asset icon would be displayed. What do you think about it?